waspc/data/Generator/templates/react-app/src/DefaultLoader.tsx

```diff
@@ -0,0 +1,37 @@
+import { styled, keyframes } from 'wasp/core/stitches.config'
```

Comment:
I'd also maybe suggest putting this file in `components/` dir so it's clear it's a component a not some top level file like `router.tsx or `index.tsx`.